Turcica.csl

```diff
@@ -0,0 +1,181 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style class="note" version="1.0" demote-non-dropping-particle="sort-only" xmlns="http://purl.org/net/xbiblio/csl">  <!-- This style was edited with the Visual CSL Editor (https://editor.citationstyles.org/visualEditor/) -->
+  <info>
+    <title>turcica.csl </title>
+    <title-short>turcica.csl </title-short>
+    <id>http://www.zotero.org/styles/revue-turcica</id>
+    <link rel="self" href="http://www.zotero.org/styles/revue-turcica"/>
+    <contributor>
```

Comment:
AFter line 7 add `<link href="https://poj.peeters-leuven.be/content.php?journal_code=TURC&url=journal" rel="documentation"/>`